kiss

```diff
@@ -1573,34 +1561,16 @@ pkg_update_repo() {
 
         am_owner "$PWD" || {
             printf 'Need "%s" to update\n' "$user"
-            set -- as_user
+            set -- 1
         }
 
         # arg1: pre-update
```

Comment:
should this be `# arg1: update`, or should 1570 be `run_hook pre-update "$#"`?

Reply:
This PR is not what the final implementation will look like. This is just to illustrate the idea. (This PR won't be merged).